test/sharness/t0130-swarm.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+test_description="Test swarm command"
+
+. lib/test-lib.sh
+
+test_init_ipfs
+test_launch_ipfs_daemon
+
+test_expect_success "'ipfs swarm addrs' succeeds" '
+	ipfs swarm addrs >swarm_addrs_out 2>swarm_addrs_err ||
+	test_fsh cat swarm_addrs_out || test_fsh cat swarm_addrs_err
+'
+
+test_swarm_addrs_output() {
+	cat "$1" |
+	while IFS='' read -r line || [[ -n $line ]]
+	do
+		grep '^Qm[0-9A-Za-z]\{44\} ([1-9][0-9]*)$' <<<"$line" >/dev/null || return
+		addr_count="$(cat <<<"$line" | sed "s/^.*(//" | sed "s/)$//")"
+		for i in $(seq "$addr_count")
+		do
+			IFS='' read -r line || [[ -n $line ]] || return
+			grep $'^\t/[^/]*/[^/]*/[^/]*/[^/]*$' <<<"$line" >/dev/null || return
+		done
+	done
```

Comment:
@jbenet

What language should I use? If everyone's gonna use their favorite language to write test helpers, the code base will inevitably become an unrecognizable mess.